crates/ruma-client-api/src/r0/account/register.rs

```diff
@@ -72,7 +72,8 @@ ruma_api! {
         ///
         /// [admin]: https://matrix.org/docs/spec/application_service/r0.1.2#server-admin-style-permissions
         #[serde(rename = "type", skip_serializing_if = "Option::is_none")]
-        pub login_type: Option<&'a LoginType>,
+        pub login_type: Option<&'a LoginType>, // TODO: should this field be gated
+        // by the app-service feature?
```

Comment:
Which appservice feature?

Reply:
Well either `appservice-api-c` or `appservice-api`. I only wondered because the field is __not__ in the client API spec at all and the only mention is in the link above (in the doc comment) saying that it can be added to that `Request` for appservice and enables a special kind of login.

Reply:
The main (only?) reason we have these feature flags is having less code for the compiler to deal with. I don't think it's worth introducing a feature flag to ruma-client-api for this one field. But if we had an appservice feature flag there already, putting this behind it would probably be good.